runtime/parachains/src/dmp.rs

```diff
@@ -102,8 +107,15 @@ pub mod pallet {
 	pub(crate) type DownwardMessageQueueHeads<T: Config> =
 		StorageMap<_, Twox64Concat, ParaId, Hash, ValueQuery>;
 
-	#[pallet::call]
-	impl<T: Config> Pallet<T> {}
+	frame_support::parameter_types! {
+		pub InitialFactor: FixedU128 = FixedU128::from_u32(1);
+	}
+
+	/// The number to multiply the base delivery fee by.
+	#[pallet::storage]
+	#[pallet::getter(fn delivery_fee_factor)]
+	pub(crate) type DeliveryFeeFactor<T: Config> =
```

Comment:
This has to live in storage, right? Since it needs to be persisted across blocks.

Reply:
Yes I think so!